packages/ui/src/machines/authenticator/actors/signIn.ts

```diff
@@ -576,9 +576,15 @@ export function signInActor({ services }: SignInMachineOptions) {
           const { phone_number_verified, email_verified } =
             event.data as Auth.FetchUserAttributesOutput;
 
-          return (
-            email_verified === 'false' && phone_number_verified === 'false'
-          );
+          // email/phone_verified is returned as a string
+          const emailNotVerified =
+            email_verified === undefined || email_verified === 'false';
+          const phoneNotVerified =
+            phone_number_verified === undefined ||
+            phone_number_verified === 'false';
+
+          // only request verification if both email and phone are not verified
```

Comment:
Think we should stick with the current behavior here at least until the migration is complete. Concerned abt updating UX while we still have moving pieces

Reply:
Yep this changes it to match the current behavior better. Previously there was a bug that if just email was verified and there was no value for `phone_number_verified` it would try to request verification